### ISSUE TITLE
Stop HTML-encoding comments when using docs.microsoft.com interface

### ISSUE THREAD

**Issue by thomps23** [MicrosoftDocs/feedback#1513]
Feedback provided via the Azure DevOps documentation--
"E.g. > becomes &gt; in the issue text. This is annoying as I then have to open the new issue on GitHub and fix the formatting."

**Comment by Andrzej-W** [comment]
I fully agree with @thomps23. The problem exists in every documentation and not only in Azure (personally I have this annoying experience with ASP.NET Core).
